pursuit.cabal

```diff
@@ -85,7 +85,7 @@ library
                  , directory                     >= 1.1        && < 1.3
                  , warp                          >= 3.0        && < 4
                  , data-default
-                 , aeson                         >= 0.6        && < 0.10
+                 , aeson                         >= 0.6        && < 0.12
```

Comment:
It will kick off a new Travis build if you do that, yeah. You can add `[ci skip]` to the commit message to stop Travis from doing a CI build, I think.